src/DefaultEditor.js

```diff
@@ -167,7 +167,7 @@ class DefaultEditor extends Component {
           </TraceAccordion>
         </Panel>
         <LayoutPanel group="Style" name={_('Layout')}>
-          <Fold name={_('Canvas')} foldIndex={0} key={0}>
```

Comment:
is `key` required here still? I was under the impression that it was required only in list settings?

Reply:
or rather, could `Panel` set the `key` just like it sets `foldIndex` ?

Reply:
`key` is a React api thing right, and usually when you render more than one component of the same type, you have to set a key on it so that react doesn't get mixed up. Usually nothing breaks when you don't, but from time to time it gives nasty bugs.

I don't want to add anything to the key prop. And the key prop could be anything, not necessarily an index.

Reply:
I don't think the `key` is need when you just enumerate components like this, just when you generate an array of them with `map()` or something. If they are required in cases like this, then this PR solves only half the problem :(

Reply:
yeah, ok, rereading docs:
https://reactjs.org/docs/lists-and-keys.html
Key is not a needed prop, I'll remove it from the Default Editor

Reply:
Hmm, despite what the docs say, I think `key` is required after all, I'm getting key warnings now.

Reply:
Probably because within `Panel` we are treating `children` as an array ... so maybe `Panel` needs to set `key` after all? It's not common practice to put `key` in a context like the `DefaultEditor`, as per the docs.